src/components/scintillation/Scintillation.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import Mount from '../mount';
+
+import './scintillation.scss';
+
+class Scintillation extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: 0,
+            show: true,
+            color: 'grey'
+        };
+    }
+
+    componentDidUpdate(prevProps, prevState) {
+        if (this.state.counter !== prevState.counter){
+            this.setState({color: '#grey'});
+        }
+        if (
+            this.state.counter !== prevState.counter &&
+            this.state.counter % 3 === 0
+        ) {
+            this.setState({color: 'green'});
+        }
+        if (
+            this.state.counter !== prevState.counter &&
+            this.state.counter % 5 === 0
+        ) {
+            this.setState({color: 'blue'});
+        }
+        if (
+            this.state.counter !== prevState.counter &&
```

Comment:
you have this condition in 4 places. Why don't you put it once in the beginning?
`if (this.state.counter !== prevState.counter) return;`